Update service_remapping_rules.md (#37203)
DataDog/documentation: content/en/tracing/services/service_remapping_rules.md

### Step 3: Name your rule and review

1. Optionally, enter a descriptive name for the remapping rule so you can identify it later.

> 1. Review and save your remapping rule. After you save your rule, _it may take a minute for it to take effect_.

Reviewer: In case it takes longer than exactly 1 minute:
```suggestion
1. Review and save your remapping rule. After you save your rule, _it may take about a minute for it to take effect_.
```

Remapping rules are applied across APM, Logs, Metrics, USM, DSM, DJM, DBM, Profiling, NPM, Live Processes, Live Containers, Kubernetes, and Events.

- **Historical data:** Changes made by remapping rules affect only telemetry ingested while a rule is active, and past data is not updated retroactively. Deleting or modifying a rule stops it from applying to new data, but does not revert names on previously ingested data.

> - **Rule order:** Service remapping rules are applied in order. Rules at the top of the rule list are applied first. A service will be remapped only by the first rule that captures it (multiple rules will not be applied to the same service).

Reviewer: ```suggestion
- **Rule order:** Service remapping rules are applied in order. Rules at the top of the rule list are applied first. A service is remapped only by the first rule that captures it (multiple rules are not applied to the same service).
```

> - **Regular expressions:** Regular expressions can be used to define new service names, but greedy quantifiers are not allowed inside the capture group.

Reviewer: Will "greedy quantifiers" be understood by readers in this context? It seems pretty specialized but I'm not familiar with these particular features. It might still help to explain the term very briefly in parentheses.

Author: I think so, I've added it because I've gotten feedback from a couple customers now that I don't mention anything about greedy quantifiers in the documentation.

Reviewer: Sounds good, thanks for confirming!
